content/account/organizations.textile

```diff
@@ -0,0 +1,193 @@
+
+<aside data-type='note'>
+<p>To start using organizations, an Enterprise account is required. "Contact":https://ably.com/contact Ably to migrate to an Enterprise account. </p>
+</aside>
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+<aside data-type='note'>
+<p>To change the primary account "contact":https://ably.com/contact Ably.</p>
+</aside>
+
+h2. Create accounts in an organization
+
+Create accounts in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Accounts*.
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h2. Isolate environments
+
+Use separate accounts within an organization to create isolated environments, such as production, staging, and development. Assign each environment a "package":/docs/pricing#packages that meets its specific needs. For example, production may need high capacity with an *Enterprise* package, while staging might use a *Standard* package. Development may operate on a *Free* package.
+
+Select the appropriate package for your account:
+
+* Open the *Account* navigation dropdown.
+** Click on *Billing*.
+** Choose the required package for the selected account and *Save billing details*.
+
+<aside data-type='note'>
+<p>Each account within an Ably Organization is assigned a unique domain. This ensures that environments such as production, staging, and development are properly isolated from one another. </p>
+</aside>
+
+h2. Manage user account access
+
+Manage user account access:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Select the user's account access you want to manage.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for required *Account* access.
+** Click *Back to Organization users to save*.
+** Add an account name and *Create account*.
+
+<aside data-type='note'>
+<p>Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably upon the next SCIM update cycle.</p>
+</aside>
+
+h3. User permissions
+
+A user's total permissions consist of:
+
+* Roles and permissions assigned directly to the user.
+* Roles and permissions granted to all groups the user is part of.
+
+h3. Group permissions with SCIM
+
+In the "Okta":#okta setup phase of the SSO and SCIM configuration, you assign users and groups to the new Okta app. The group you created in Okta (your identity provider) is available to view and for user access management within Ably:
```

Comment:
This is a little odd that we reference a section further down the page in an earlier one in the sense that you should have already done it. I think they should switch places.